tests/test_importers.py

```diff
@@ -1033,5 +1035,38 @@ def test_skip_file(self, resolver_prompt, menu_prompt, file_processor_prompt):
             assert len(self.store.session.query(self.store.db_classes.Comment).all()) == comments
 
 
+def test_changes_table_entry_contains_version():
```

Comment:
Test for version looks good Robin. Would it be possible to have another test to check `__build_timestamp__` being applied?

Reply:
Unfortunately that would be fairly difficult - as the build timestamp is only set when we do a full release. That means that when we're running the tests on the development version, `__build_timestamp__` will always be None.

Reply:
Aah, as it happens I'm going to make a new release today -  so I can offer a manual test to verify it.

Reply:
No problem Robin. If you're happy to test as part of your release Ian, that would be great.